### PR TITLE
Full website with navigation

### PR DESCRIPTION
**Pull request by Shaheryar0054**
- Install react router dom
- Added 3 different pages for navigation
- Added some styling
- Used React Router to make the routes work.

### CODE REVIEW

File: src/components/Rout.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      <h1>Math Magicians</h1>
+      <h2>Welcome to our page!</h2>
+      <p>{text}</p>
+    </div>
+  );
+}
+function CalculatorPage() {
+  return (
+    <div className="calculator-content">
+      <h1>Math Magicians</h1>
```
Comment (codepantha) [inline src/components/Rout.js]: - [ ] Please let's ensure that our design follows the layout guideline of the wireframe correctly. Please notice that in the wireframe the calculator page has "Let's do some math" on the left and the Calculator on the right taking up the rest of the available screen.
### Wireframe guideline
image

### Your implementation
image

- [ ] The way you have set up all 3 component pages (`<Home />`, `<CalculatorPage />` and `<Quote />`) in the same `Rout.js` file works quite alright but it is neither professional nor clean, and doesn't make your code easy to read. You can see the potential of this file being too long if you were to do something more complex, but the added disadvantage of difficulty while debugging. Please extract all 3 pages into their own separate individual files 👍

File: src/components/Rout.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      <Calculator />
```
Comment (codepantha) [inline src/components/Rout.js]: - [ ] [OPTIONAL] As this is not a strict requirement for this milestone and should have been caught earlier, I'm marking it as optional.
There is an error on the console which clearly shows that you have provided a `value` prop to an input field without an `onChange` handler. This input field in question is what you used as your display in the `<Calculator />` component 😄 Please consider changing it to a div instead?
**HINT:**
Something like this won't be too bad 😉 
```javascript
<div className="input">{state.next || state.total || '0'}</div>
```
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @Shaheryar0054 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitFlow
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly remove the link at the bottom of your README which is a part of the template as it doesn't make your README look very professional at the moment 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please try to keep your commit messages in the imperative mood. For instance, instead of "Added nav menu on header section" you could say "Add nav menu on header section" 😎 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @Shaheryar0054 , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  All pages added and can visit each page from any page
:heavy_check_mark: App looks like given wireframe
:heavy_check_mark: Descriptive ReadMe file and PR

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions

 [OPTIONAL ] I am marking these requirements optional because the previous reviewer missed them. Kindly consider doing the changes before merging this PR.
- When I click on your Live demo link, it redirects to an error 404 page. It is not professional to push broken links to Github. If you don't yet have a working deployed link, you can use the text `Coming Soon ... 
`
Screenshot from 2023-03-01 07-48-57

- There' s some text below the MIT license link that you can consider removing it. It is provided as a guide to help in getting correctly the license link.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**